app/src/main/java/com/kks/nimblesurveyjetpackcompose/model/SurveyQuestion.kt

```diff
@@ -1,5 +1,22 @@
 package com.kks.nimblesurveyjetpackcompose.model
 
 data class SurveyQuestion(
-    val title: String
+    val id: String,
+    val title: String,
+    val displayOrder: Int,
+    val shortText: String,
+    val pick: String,
+    val questionDisplayType: QuestionDisplayType,
+    var answers: List<SurveyAnswer>
 )
+
+enum class QuestionDisplayType(val typeValue: String) {
+    NONE("none"), INTRO("intro"), DROP_DROWN("dropdown")
```

Comment:
I think dropdown is a single word. 💦 
```suggestion
    NONE("none"), INTRO("intro"), DROPDOWN("dropdown")
```